pkg/project/apiserver/admission/nodeenv/admission_test.go

```diff
@@ -25,6 +27,7 @@ func TestPodAdmission(t *testing.T) {
 	projectStore := projectcache.NewCacheStore(cache.IndexFuncToKeyFuncAdapter(cache.MetaNamespaceIndexFunc))
 	projectStore.Add(project)
 
+	externalMockClientset := fakev1.NewSimpleClientset()
 	mockClientset := fake.NewSimpleClientset()
```

Comment:
this looks questionable.  What is it for?

Reply:
int the line below `handler := &podNodeEnvironment{client: ...}`, `podNodeEnvironment` requires an internal clientset